README.md

```diff
@@ -5,6 +5,21 @@
 
 Enable email via postfix.
 
+## relayhost
+Options:
+* `outbound-relays.ncsa.illinois.edu` (default)
+  * Useful for hosts in NCSA Private IP space (172.24.0.0/13).
+* `outbound-relays.techservices.illinois.edu`
+  * Recommended for hosts in NCSA Public IP space (141.142.0.0/16)
+    * Note: must also have a public hostname (e.g. *.illinois.edu) and
+      that hostname must be set in `myhostname` and `myorigin`. The default
+      settings work fine for the most common case when a node has just one
+      hostname and one IP, but more care is needed when a node has more than one.
+
+See also:
+* [University of Illinois IP Spaces](https://go.illinois.edu/ipspace)
+* [Email, Unauthenticated SMTP for campus](https://answers.uillinois.edu/illinois/47888)
+
 ## root_mail_target
 
 The default root email address is devnull@ncsa.illinois.edu. devnull@ncsa.illinois.edu is an actual mail alias setup on NCSA’s pop server that just drops the message. This will have to be changed via hiera data in order to have root mail sent to the appropriate place.
```

Comment:
This default address may need to change after legacy NCSA mail servers are decommissioned in the coming months.

Reply:
That address is being kept. It will route to nobody@illinois.edu and get dropped there.